src/internal_modules/roc_address/print_supported.cpp

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2019 Roc Streaming authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include "roc_address/print_supported.h"
+#include "roc_address/protocol_map.h"
+#include "roc_core/log.h"
+#include "roc_core/printer.h"
+
+namespace roc {
+namespace address {
+
+namespace {
+
+enum { ArraySize = 100, LineSize = 70 };
```

Comment:
Please remove ArraySize, it's not used.

BTW could you also remove ArraySize from src/internal_modules/roc_sndio/print_supported.cpp, from where you copied it? It's not used there too.